environment/postInstall

```diff
@@ -1,5 +1,16 @@
 #!/usr/bin/env bash
 set -e
 
+# install spikeinterface extras
+pip install --no-cache-dir --ignore-installed spikeinterface[full,widgets]==0.98.0
```

Comment:
I think this should be fine. In my current pr in aind-data-transfer, I pinned the spikeinterface version <0.98.0 because the ephys reader unit tests were failing. It looks like the compression worked when I ran it against a few small data sets though.